todo_app/templates/index.html

```diff
@@ -12,7 +12,19 @@ <h1 class="display-4">To-Do App</h1>
       <h2>Items</h2>
       <ul class="list-group mb-4">
         <!-- List the todo items here -->
+        {% for value in index %}
+          <li>{{value.title}}</li>
+        {% endfor %}
       </ul>
+      <p>
+        <form action="/new" method="POST" class="new-item">
+          <div class="new-item">
```

Comment:
An element's "class" is mostly used to control styling (via css). You don't have any styling rules for "new-item" elements, so there's no reason to stick that class on these.